opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -216,7 +216,7 @@ def __init__(
         self,
         name: str,
         context: trace_api.SpanContext,
-        parent: trace_api.ParentSpan = None,
+        parent: Optional[trace_api.SpanContext] = None,
```

Comment:
The docstring for the arg needs to change.

Reply:
Actually I just took a look, looks okay as is?

> parent: This span's parent, may be a `SpanContext` if the parent is remote, null if this is a root span

Reply:
actually I see the ambiguity there, I'll clear that up.